pages/index.tsx

```diff
@@ -1,20 +1,25 @@
 import type { InferGetStaticPropsType } from 'next';
 
 import { closeConnection, openConnection } from '../api-helpers/db';
+import { ArticleTile } from '../components/ArticleTile/ArticleTile';
 import { Layout } from '../components/Layout';
+import { createExcerpt } from '../utils/excerpt-utils';
+
+export type HomePageProps = InferGetStaticPropsType<typeof getStaticProps>;
 
-type HomePageProps = InferGetStaticPropsType<typeof getStaticProps>;
 export default function HomePage({ blogs }: HomePageProps) {
   return (
     <Layout>
       <h1>Siema!</h1>
-      <ul>
+      <ul style={{ listStyleType: 'none', padding: 0, margin: 0 }}>
         {blogs.map((blog) => (
           <li key={blog.id}>
             {blog.name}
-            <ul>
+            <ul style={{ listStyleType: 'none', padding: 0, margin: 0 }}>
```

Comment:
Nie wiem czy to tymczasowe rozwiązanie, ale jeśli nie to chyba stworzyłbym dla tych dwóch list klasę

Reply:
jest task na tę listę